api/client/nic.cc

```diff
@@ -417,6 +418,9 @@ int NicAddOptions::Parse(int argc, char **argv) {
             vni = string(argv[i + 1]);
         else if (string(argv[i]) == "--bypass-filtering")
             bypass_filtering = "true";
+        else if ((string(argv[i]) == "--packet-trace") ||
+                 (string(argv[i]) == "-t"))
+            packet_trace = "packet_trace: " + std::string(argv[i + 1]);
```

Comment:
you don't check what's inside argv[i + 1]
example: 
Here I can use `--packet_trace toto` and client will say nothing